smart-employee-agent/hr-server/config.py

```
@@ -0,0 +1,33 @@
+    raise ValueError(
+        "Missing required environment variables: AUTH_ISSUER, CLIENT_ID, or JWKS_URL"
+    )
+
+ALLOWED_ORIGINS = [
+    o.strip()
+    for o in os.getenv("ALLOWED_ORIGINS", "http://localhost:3000,http://127.0.0.1:3000").split(",")
+    if o.strip()
+]
+
+PORT = int(os.getenv("HR_SERVER_PORT", os.getenv("PORT", "8000")))
+HOST = os.getenv("HR_SERVER_HOST", "0.0.0.0")
```

Ruff (0.15.11)

[error] Possible binding to all interfaces

(S104)